packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -25,26 +25,26 @@ Client-based rate limiting can be helpful in the following scenarios:
 
 ### How it works
 
-The best way to describe how client-based rate limiting works is to provide some example use cases.
+The best way to describe how client-based rate limit works is to provide some example use cases.
 
 #### Client-based isolation for a unique IP address
 
-Bob and Alice are working from home and have distinct IP addresses. Both Bob and Alice try to sign in to their company's portal application (clientID: portal123) at the same time. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate-limiting framework creates a unique per minute request quota for both Bob and Alice using their IP address and the OAuth client ID that they are trying to access.
+Bob and Alice are working from home and have distinct IP addresses. Both Bob and Alice try to sign in to their company's portal application (clientID: portal123) at the same time. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate-limit framework creates a unique per minute request quota for both Bob and Alice using their IP address and the OAuth client ID that they are trying to access.
 
 * Bob: (IP1 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
 * Alice: (IP2 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
 
 Let's assume the org-wide quota for the `/authorize` endpoint is a total of 2,000 requests per minute on Custom Authorization Servers. Bob decides to run a batch job that triggers about 2,000 `/authorize` requests per minute.
 
-Without the client-based rate-limiting framework, Bob consumes all of the total allowed requests per minute (2,000 requests per minute). This results in HTTP 429 errors for both Bob and Alice, which makes the application inaccessible for everyone.
+Without the client-based rate-limit framework, Bob consumes all of the total allowed requests per minute (2,000 requests per minute). This results in HTTP 429 errors for both Bob and Alice, which makes the application inaccessible for everyone.
```

Comment:
"rate-limit" -> "rate limit"

Reply:
Fixed. (And also fixed the same issue in the next paragraph.)